rskj-core/src/main/java/org/ethereum/core/TransactionExecutor.java

```diff
@@ -240,9 +234,7 @@ public void execute() {
             BigInteger txGasCost = toBI(tx.getGasPrice()).multiply(txGasLimit);
             track.addBalance(tx.getSender().getBytes(), txGasCost.negate());
 
-            if (logger.isInfoEnabled()) {
-                logger.info("Paying: txGasCost: [{}], gasPrice: [{}], gasLimit: [{}]", txGasCost, toBI(tx.getGasPrice()), txGasLimit);
-            }
+            logger.trace("Paying: txGasCost: [{}], gasPrice: [{}], gasLimit: [{}]", txGasCost, toBI(tx.getGasPrice()), txGasLimit);
```

Comment:
Maybe we should keep a `.isTraceEnabled()` in place, this way the `toBI()` will not be invoked when the TRACE level is not enabled.

Reply:
toBi() shouldn't be expensive but we can keep the check